ui/app/styles/components/box-label.scss

```diff
@@ -9,12 +9,8 @@ label.box-label {
 
 .box-label {
   @extend .box;
-  // @extend .is-centered;
-  // @extend .is-gapless;
-  justify-content: center;
```

Comment:
We had added these in to replace the two extended helpers. This CSS is now no longer needed.